ert/storage/_storage.py

```diff
@@ -135,46 +135,93 @@ async def _post_to_server_async(
     return resp
 
 
-async def add_record(url: str, record: Any) -> None:
+def _get_content_type(record_type: ert.data.RecordType) -> str:
+    content_type = (
+        ContentTypeEnum.OCTET_STREAM
+        if record_type == ert.data.RecordType.BYTES
+        else ContentTypeEnum.CSV
+    )
+    return content_type.value
```

Comment:
If `ContentTypeEnum` would simply be a class this could be written as:

```python 
def _get_content_type(record_type: ert.data.RecordType) -> str:
    if record_type == ert.data.RecordType.BYTES:
        return ContentTypeEnum.OCTET_STREAM
    return ContentTypeEnum.CSV

```

you don't really have to deal with all that content_type.value stuff that is a bit anying. 

also adding a function for this that is used in only one place maybe it is a bit much...

Reply:
removed